src/dropdown.styles.ts

```diff
@@ -134,16 +134,15 @@ export default [
       padding: 0;
       position: absolute;
 
-      &.no-results {
+      &.optionless {
```

Comment:
"optionless" because it reads more like a state modification and to accommodate both the `hasNoAvailableOptions` and `hasNoMatchingOptions` cases.